todo-app/src/components/TodoList.vue

```diff
@@ -43,11 +50,15 @@ function saveChanges(editedTodo: Todo, todoId: number) {
   emit('updateTodo', editedTodo, todoId);
 }
 
-function setIsEditingTrue(todo: Todo) {
-  emit('setIsEditingTrue', todo.id);
+function setIsEditingTrue(todoId: number) {
+  emit('setIsEditingTrue', todoId);
 }
 
```

Comment:
when emiting you can pass more than one argument: emit('setEditState', todoId, true) or emit('setEditState', todoId, false)
<img width="704" alt="image" src="https://github.com/nykami/todo-app/assets/81409985/3c6ff571-d540-4ce4-852c-33e611e3eaeb">